tests/cli/test_buildspec.py

```diff
@@ -231,6 +232,13 @@ def test_buildspec_find_invalid():
     except SystemExit:
         pass
 
+    try:
+        cache.print_invalid_buildspecs(
+            error=False, terse=False, header=False, row_count=True
+        )
+    except SystemExit:
+        pass
+
```

Comment:
i think you can replace this section with. Its the same thing but using `try/except` with a `pass` is not a good practice. We know this method will raise `sys.exit`. I do see other parts of this code-base uses `try/except` which could also be changed but thats not relevant for this PR.

```
with pytest.raises (SystemExit):
           cache.print_invalid_buildspecs(
            error=False, terse=False, header=False, row_count=True
        )
```